prog/test/vm_group.rb

```diff
@@ -135,6 +135,46 @@ def self.assemble(storage_encrypted: true, test_reboot: true)
       nap 5
     end
 
+    hop_verify_purge
+  end
+
+  def verify_vm_dir_purged
+    sshable = vm_host.sshable
+    vm_dir_content = sshable.cmd("sudo ls -1 /vm").split("\n")
```

Comment:
Since we run VM group tests and GitHub actions tests in parallel, even if this group doesn't have an active VM, others might. We might move this test to just before the Hetzner server is destroyed